packages/discord.js-utilities/src/lib/PromptMessage.ts

```diff
@@ -0,0 +1,150 @@
+import type {
+	Message,
+	User,
+	TextChannel,
+	NewsChannel,
+	APIMessageContentResolvable,
+	MessageOptions,
+	MessageAdditions,
+	DMChannel,
+	CollectorFilter,
+	EmojiIdentifierResolvable,
+	MessageReaction
+} from 'discord.js';
+
+/**
+ * This is a [[PromptMessage]], a utility to send a message with Yes/No promt.
+ * You must either use this class directly or extend it.
+ *
+ * [[PromptMessage]] uses actions, these are essentially reaction emojis, when triggered run the said action.
+ * You can utilize your own actions, or you can use the [[PromptMessage.defaultPrompts]].
+ * [[PromptMessage.defaultPrompts]] is also static so you can modify these directly.
+ *
+ * @example
+ * ```typescript
+ * const handler = new PromptMessage('Are you sure you want to continue?');
+ * const confirmed = handler.run();
+ * ```
+ *
+ * @example
+ * ```typescript
+ * const handler = new PromptMessage('Are you happy or sad?', {
+ * 		confirm: '🙂',
+ *		cancel: '🙁'
+ * });
+ * const confirmed = handler.run();
```

Comment:
```suggestion
 * const confirmed = handler.run(channel);
```

Reply:
Will update to `handler.run(channel, author)` for both examples.